packages/documentation/src/pages/platform/tools/feature-flags.mdx

```diff
@@ -55,7 +55,7 @@ localhost: false
   - Making code that's easy to delete ensures that deprecated code is quickly removed once a feature is released.
 - Making features or applications available in production should done only as needed for testing with users.
 
-## Feature flags for Metalsmith code
+## CMS feature flags
```

Comment:
If you haven't, you might want to search for links to the old header (something like `#feature-flags-for-metalsmith-code`) on other pages.

Reply:
We're calling these "query config flags" now. Open to other suggestions: https://docs.google.com/a/adhocteam.us/document/d/1ESEH7P8BYFZdqMEd9ePzN3L8stuH8E8FwBViR8tZvos/edit?disco=AAAADb64FVE&usp=comment_email_discussion&usp_dm=false&ts=5d69a21c

Reply:
"feature flags" is reserved for the flipper toggles.